website/docs/api/misc/logger.md

```diff
@@ -0,0 +1,67 @@
+---
+sidebar_position: 1
+title: '📦 logger'
+slug: '/api/plugins/@docusaurus/logger'
+---
+
+An encapsulated logger for semantically formatting console messages.
+
+Authors of packages in the Docusaurus ecosystem are encouraged to use this package to provide unified log formats.
+
+## APIs
+
+It exports a single object as default export: `logger`. `logger` has the following properties:
+
+- Some useful colors.
+  - `red`
+  - `yellow`
+  - `green`
+  - `bold`
+  - `dim`
+- Formatters. These functions all have the signature `(msg: unknown) => string`. Note that their implementations are not guaranteed. You should only care about their semantics.
+  - `path`: formats a file path.
+  - `url`: formats a URL.
+  - `name`: formats an identifier.
+  - `code`: formats a code snippet.
+  - `subdue`: subdues the text.
+  - `num`: formats a number.
+- The `interpolate` function. It is a template literal tag. The syntax can be found below.
+- Logging functions. All logging functions can both be used as normal functions (similar to the `console.log` family, but only accepts one parameter) or template literal tags.
+  - `info`: prints information.
+  - `warn`: prints a warning that should be payed attention to.
+  - `error`: prints an error (not necessarily halting the program) that signals significant problems.
+  - `success`: prints a success message.
+
+:::caution A word on the `error` formatter
```

Comment:
not a big deal but these admonition titles might end up causing problems in Crowdin/i18n as translators end up translating the admonition key